src/i18n/en-US/index.ts

```diff
@@ -32,7 +33,8 @@ const enUS = {
   intake,
   privacyPolicy,
   taskList,
-  termsAndConditions
+  termsAndConditions,
+  survey
```

Comment:
I'm not sure I would add it's own namespace for this. The main reason is because I don't think we'd have more than a handful of translations for this. It doesn't seem "big" enough to warrant its own namespace. It can probably be added to general since it's application wide.

Reply:
I didn't realize there was a general namespace for translations. I agree with you. Actually the text for the links is already in general although it needs to be changed slightly